data_collection/processing/channel_detection.rst

```diff
@@ -19,6 +19,10 @@ Channel detection approach
 
 This section describes how Piwik PRO Analytics detects a channel from the tracked event. The order this section is arranged by represents the order of detection. Detection stops when the conditions of the currently processed step are met.
 
+    .. note::
+        Despite detected channel, dimensions ``source``, ``medium``, ``referrer_keyword`` and ``keyword``  can be overwritten by passed in url campaign parameters. Such parameters are called ``unnamed campaign parameters``, as they miss ``campaign name`` parameter, which would assign such event to the ``campaign`` category.
```

Comment:
```suggestion
        Despite detected channel, dimensions ``source``, ``medium``, ``referrer_keyword`` and ``keyword``  can be overwritten by URL defined campaign parameters. Such parameters are called ``unnamed campaign parameters``, as they miss ``campaign name`` parameter (otherwise such event would be processed as ``Campaign`` type).
```